fragments/master.sh

```diff
@@ -211,9 +211,55 @@ start_k8s(){
 
 }
 
+pull_addons_image(){
```

Comment:
hmm..attempt to paralyze wait for api server and pulling image? However, pull should be fast. Only a few bytes, I assume. Is it worth the extra code?